### PR TITLE
added voxed theme

### PR DESCRIPTION
**Pull request by rojebd**
### Added [Voxed](https://github.com/rojebd/voxed) 
 a custom theme  I made for helix

Screenshots:
image
image
image

### CODE REVIEW

File: runtime/themes/voxed.toml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+"diagnostic.info" = { underline = { color = "delta", style = "curl" } }
+"diagnostic.warning" = { underline = { color = "lightning", style = "curl" } }
+"diagnostic.error" = { underline = { color = "apricot", style = "curl" } }
+
+warning = "bpink"
+error = "bsienna"
+info = "maize"
+hint = "tan"
+
+[palette]
```
Comment (the-mikedavis) [inline runtime/themes/voxed.toml:95]: The palette doesn't define many of the colors that are used in the theme, for example `apricot`, `lavendar` or `sorocco`

File: runtime/themes/voxed.toml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+"ui.popup" = { fg = "bgray", bg = "" }
+"ui.window" = { fg = "white" }
+"ui.help" = { bg = "#3f4047", fg = "light-grey" }
+
+"ui.text" = { fg = "lavender" }
+"ui.text.focus" = { fg = "maize", bg = "bgray" }
+"ui.text.inactive" = "sirocco"
+"ui.virtual" = { fg = "comet" }
+
+"ui.virtual.indent-guide" = { fg = "comet" }
```
Comment (the-mikedavis) [inline runtime/themes/voxed.toml:67]: You may want to define rulers as a background color, for example `"ui.virtual.ruler" = { bg = "light-gray" }`

File: runtime/themes/voxed.toml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+bsienna = "#D5896F"
+bpink = "#FF5964"
+maize = "#FFE74C"
+bgray = "#8c8681"
+sglow = "#FFCF56"
+status = "#15616D"
+status-two = "#3879A1"
+redish = "#E76B74"
+light-grey = "#b7afa8"
+bgray-two = "#706b68"
```
Comment (the-mikedavis) [inline runtime/themes/voxed.toml:109]: It looks like there's mixed usage of "grey" and "gray"

Comment (rojebd) [reply runtime/themes/voxed.toml]: okay fixed that

File: runtime/themes/voxed.toml (hunk 2 above)
Comment (rojebd) [reply runtime/themes/voxed.toml]: applied

File: runtime/themes/voxed.toml [hunk 4]
```diff
@@ -0,0 +1,110 @@
+attribute = "buff"
```
Comment (the-mikedavis) [inline runtime/themes/voxed.toml:1]: buff isn't defined in the palette

File: runtime/themes/voxed.toml [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+# NOTE: bg is empty so it is transparent on the terminal
+# if you want the bg to not be transparent rename give bg
+# the same value as fg and remove fg :)
+"ui.background" = { bg = "#25262B" }
+"ui.background.separator" = { fg = "sglow" }
+"ui.linenr" = { fg = "light-grey", modifiers = ["italic"] }
+"ui.linenr.selected" = { fg = "bpink", modifiers = ["bold"] }
+"ui.statusline" = { fg = "status-two", bg = "light-grey", modifiers = ["bold", "dim"] }
+"ui.statusline.inactive" = { fg = "black", bg = "bgrey-two" }
+"ui.popup" = { fg = "bgrey", bg = "" }
```
Comment (the-mikedavis) [inline runtime/themes/voxed.toml:58]:
```suggestion
"ui.popup" = { fg = "bgrey" }
```

File: runtime/themes/voxed.toml [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+"markup.link.url" = { fg = "sglow", modifiers = ["underlined"] }
+"markup.link.text" = "greenish"
+"markup.raw" = "light-grey"
+
+"diff.plus" = "#7DDF64"
+"diff.minus" = "#F22B29"
+"diff.delta" = "#6f44f0"
+
+# TODO: diferentiate doc comment
+# concat (ERROR) @error.syntax and "MISSING ;" selectors for errors
```
Comment (the-mikedavis) [inline runtime/themes/voxed.toml:47]: Could you remove the comments like this that are copied from the default theme?

File: runtime/themes/voxed.toml (hunk 4 above)
Comment (rojebd) [reply runtime/themes/voxed.toml:1]: sorry i forgot i defined it now

**Review by the-mikedavis — DISMISSED** [review]

**Pull request merged**